suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -951,3 +951,26 @@ export const getTxHeaderSymbol = (transaction: WalletAccountTransaction) => {
 
     return symbol;
 };
+
+export const groupTransactionsBySymbol = (txs: WalletAccountTransaction[]) => {
```

Comment:
nitpick: Wouldn't this be worth a simple unit test?

Reply:
Was thinking about it and a very few functions from this file have tests written for them but I should probably not follow this pattern 😃

Reply:
Done